tests/test_loaders.py

```diff
@@ -67,15 +67,31 @@ def loader_module():
     return loaders.PluginLoader(
         content_type=constants.ContentType.MODULE,
         rel_path='plugins/modules/my_sample_module.py',
-        root='/tmp/tmpiskt5e2n')
+        root='/tmp_placeholder/tmp_placeholder/ansible_collections/my_ns/my_collection')
 
 
 @pytest.fixture
 def loader_role():
     return loaders.RoleLoader(
         content_type=constants.ContentType.ROLE,
         rel_path='roles/my_sample_role',
-        root=None)
+        root='/tmp_placeholder/tmp_placeholder/ansible_collections/my_ns/my_collection')
+
+
+@pytest.fixture
+def loader_module_subdirs():
+    return loaders.PluginLoader(
+        content_type=constants.ContentType.MODULE,
+        rel_path='plugins/modules/subdir1/subdir2/my_sample_module.py',
+        root='/tmp_placeholder/tmp_placeholder/ansible_collections/my_ns/my_collection')
```

Comment:
Are these paths never actually used?

Reply:
they are used in the `.load` method where it calls class variables to string parse `self.root`, otherwise i was thinking of just setting to None

Reply:
So the stuff that uses it ends up getting mocked out in the tests?

Reply:
the tests either use the value defined here, or alter it if needed for the particular test